llama_stack/templates/dependencies.json

```diff
@@ -642,6 +602,46 @@
     "sentence-transformers --no-deps",
     "torch torchvision --index-url https://download.pytorch.org/whl/cpu"
   ],
+  "dev": [
```

Comment:
shouldnt this be starter as well?

Reply:
yeah there was a bug which pre-commit caught, fixed that